backend/ecommerce/models.py

```diff
@@ -1,7 +1,18 @@
 from django.db import models
 
-# Create your models here.
 
+class Product(models.Model):
+    name = models.CharField(max_length=255)
+    price = models.IntegerField()
+    discount = models.IntegerField(default=0)
+    stock = models.IntegerField(default=0)
 
-class Hii(models.Model):
-    name = models.IntegerField()
+    class Type(models.IntegerChoices):
```

Comment:
User models.TextChoices
refer [this](https://docs.djangoproject.com/en/3.1/ref/models/fields/#enumeration-types)